core/src/main/java/com/netflix/conductor/auth/AuthManager.java

```diff
@@ -209,4 +230,22 @@ public JsonQuery load(@Nonnull String query) throws JsonQueryException {
 		};
 		return CacheBuilder.newBuilder().expireAfterWrite(1, TimeUnit.HOURS).maximumSize(1000).build(loader);
 	}
+
+	private void cacheAuthData(AuthResponse authResponse) {
+		Map<String, Object> decoded = decode(authResponse.getAccessToken());
+		Object expValue = decoded.get("exp");
```

Comment:
do we need to decode token as expire time already in auth response and it is `expires_in`?